pgjdbc/src/main/java/org/postgresql/util/DriverInfo.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2017, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.util;
+
+/**
+ * Utility class with constants of Driver information.
+ */
+public final class DriverInfo {
+
+  private DriverInfo() {
+  }
+
+  public static final String DRIVER_NAME = "PostgreSQL JDBC Driver";
+  public static final String DRIVER_VERSION = "/*$mvn.project.property.parsedversion.osgiversion$*/";
+  public static final String DRIVER_FULL_NAME = DRIVER_NAME + " " + DRIVER_VERSION;
+
+  public static final int MAJOR_VERSION = /*$mvn.project.property.parsedversion.majorversion+";"$*//*-*/42;
+  public static final int MINOR_VERSION = /*$mvn.project.property.parsedversion.minorversion+";"$*//*-*/0;
+  public static final int PATCH_VERSION = /*$mvn.project.property.parsedversion.incrementalversion+";"$*//*-*/0;
+
+  public static final String JDBC_VERSION = "/*$mvn.project.property.jdbc.specification.version$*/";
+  public static final int JDBC_MAJOR_VERSION = Integer.parseInt(JDBC_VERSION.split("\\.")[0]);
+  public static final int JDBC_MINOR_VERSION = Integer.parseInt(JDBC_VERSION.split("\\.")[1]);
```

Comment:
in poms there is also `jdbc.specification.version.nodot` - might be use it and int math operations instead?